pages/career/applyjob/[jobID].page.js

```diff
@@ -0,0 +1,468 @@
+import React, { useState, useEffect } from "react";
+import { useRouter } from "next/router";
+import { useJobContext } from "../../../context/JobContext";
+import Button from "../../../components/UI/Button/Button";
+import { toast, ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+function JobApplicationForm() {
+  const router = useRouter();
+  const { jobDetails, particularJob, setParticularJob } = useJobContext();
+  const { jobID } = router.query; // Get jobID from the query parameters
+  const [captcha, setCaptcha] = useState("");
+  const [captchaError, setCaptchaError] = useState("");
+
+  const [formData, setFormData] = useState({
+    resume: "",
+    firstName: "",
+    middleName: "",
+    lastName: "",
+    email: "",
+    mobilePhone: "",
+    experienceYears: "",
+    experienceMonths: "",
+    currentSalary: "",
+    expectedSalary: "",
+    availableToJoin: "",
+    currentLocation: "",
+    notes: "",
+    rfa: "",
+    linkedin: "",
+    portfolio: "",
+    github: "",
+    consent: false,
+  });
```

Comment:
this form data is in job application form? and component almost code matched with this job id page code?